controllers/vaultstaticsecret_controller_test.go

```diff
@@ -91,7 +91,7 @@ func Test_makeK8sSecret(t *testing.T) {
 				},
 			},
 			expectedK8sSecret: nil,
-			expectedError:     fmt.Errorf(`failed to marshal key "password" from Vault secret: json: unsupported type: chan int`),
```

Comment:
The extra info in this error message seems like it could be useful, did we lose it for a reason?